### ISSUE TITLE
[BUG] (ManjaroLinux 16.06.1 Daniella) Error 520: Web server is returning an unknown error

### ISSUE THREAD

**Issue by valrossenOliver** [crmarsh/discord-linux-bugs#83]
Hello, first time I started Discord it started off pretty good. Normal bootup except, never got past the randomized funny/geeky startup text (example: Constructing aditional pylons...). It kept loading forever. Second restart and onwards only gives me 520. I tried reinstalling but still same problem. I've tried to connect to Discord using windows 10 afterwards and sucessfully coneccted, but nothing on manjaro.

Here's a screenshot of most info I could think of you needing: http://i.imgur.com/cgpGns6.png

OBS: The 'Discord' file is a .desktop file I created myself. For the purpose of the report being as vanilla as possible I launched the normal executable file.
Here's my Desktop Entry (if it is the thing that broke it [not likely]):
```
[Desktop Entry]

# The type as listed above
Type=Application

# The version of the desktop entry specification to which this file complies
Version=1.0

# The name of the application
Name=Discord

# A comment which can/will be used as a tooltip
Comment=A client to client or client to server chat based program.

# The path to the folder in which the executable is run
Path=/home/oliver/.discord

# The executable of the application.
Exec=/home/oliver/.discord/DiscordCanary

# The name of the icon that will be used to display this entry
Icon=/home/oliver/.discord/discord.png

# Describes whether this application needs to be run in a terminal or not
Terminal=false

# Describes the categories in which this entry should be shown
Categories=Accessories;
```

**Comment by valrossenOliver** [comment]
Update... after trying to run the application with a few different arguments I gave up yesterday, to try once again today with no luck. But before I went to the city I once more tried, no success and left computer... and when I come back, and ONCE AGAIN try to run it... it works? Huh? Anyway.. it starts again. No idea what caused Discord Canary not to launch during these hours (20+) only on my Manjaro... but it works now.
